docs/langgraph-deployments-and-revisions.md

```diff
@@ -0,0 +1,102 @@
+# LangGraph Deployments and Revisions
+
+This document is the single source of truth for understanding LangGraph Cloud deployment and revision statuses.
+
+## Overview
+
+LangGraph Cloud has two distinct status types that are often confused:
+
+| Entity | Status Enum | Terminal Success | Where Visible |
+|--------|-------------|-----------------|---------------|
+| **Deployment** | `DeploymentStatus` | `Ready` | `langstar graph list` |
+| **Revision** | `RevisionStatus` | `Deployed` | Revision polling, API responses |
+
+## Deployment vs Revision
+
+A **Deployment** is the container for your LangGraph application. It has an ID, name, and configuration.
+
+A **Revision** is a specific build/version of the deployment. Each time you create or patch a deployment, a new revision is created that goes through build and deploy stages.
+
+```
+Deployment (e.g., "pr-integration-test")
+├── Revision 1 (initial) → Deployed
+├── Revision 2 (after patch) → Deployed
+└── Revision 3 (latest) → Building...
+```
+
+## DeploymentStatus
+
+The overall status of a deployment resource.
+
+| Status | Description |
+|--------|-------------|
+| `AwaitingDatabase` | Database provisioning in progress |
+| `Ready` | **Terminal success** - Deployment is operational |
+| `Unused` | Deployment is inactive |
+| `AwaitingDelete` | Deletion in progress |
+| `Unknown` | Status cannot be determined |
+
+**API Value**: `SCREAMING_SNAKE_CASE` (e.g., `"READY"`, `"AWAITING_DATABASE"`)
+
+**Rust Enum**: `DeploymentStatus::Ready`
+
+## RevisionStatus
+
+The build/deploy status of a specific revision.
+
+| Status | Description |
+|--------|-------------|
+| `Queued` | Waiting in build queue |
+| `Building` | Docker image build in progress |
+| `BuildSucceeded` | Build completed, awaiting deployment |
+| `BuildFailed` | Build failed (check logs) |
+| `AwaitingDeploy` | Waiting to be deployed |
+| `Deploying` | Container deployment in progress |
+| `Deployed` | **Terminal success** - Revision is live |
+| `DeployFailed` | Deployment failed (check logs) |
+| `Cancelled` | Build/deploy was cancelled |
+| `Unknown` | Status cannot be determined |
+
+**API Value**: `SCREAMING_SNAKE_CASE` (e.g., `"DEPLOYED"`, `"BUILDING"`)
+
+**Rust Enum**: `RevisionStatus::Deployed`
+
+## Typical Revision Lifecycle
+
+```
+Queued → Building → BuildSucceeded → AwaitingDeploy → Deploying → Deployed
+                 ↘ BuildFailed
+                                                              ↘ DeployFailed
+```
+
+## Test Fixtures Behavior
```

Comment:
This `Test Fixtures Behavior` heading doesn't belong here. The purpose of this document is for users of the langstar SDK and CLI to understand the business concepts abstracted by the library. It's a good idea to have a heading similar to this, but that should likely be inside of the test documentation.

Reply:
Fixed in commit 60c9e27: Removed the "Test Fixtures Behavior" section from the user-facing documentation.